server/nlp/nlp.py

```diff
@@ -79,6 +85,22 @@ def similarity(doc):
         for topic, summaries in NLProcessor.__sim_topic_summaries.items()}
         return max(topic_sims.values())
 
+    @staticmethod
+    def cluster_similarity(doc):
+        best_similarity = NLProcessor.__get_word_vectors().wmdistance(NLProcessor.__normal_tokens(doc, stem=False), list(NLProcessor.__sim_summary_clusters.keys())[0])
+        best_key = list(NLProcessor.__sim_summary_clusters.keys())[0]
+        for cluster_key in list(NLProcessor.__sim_summary_clusters.keys()):
+            current_similarity = NLProcessor.__get_word_vectors().wmdistance(NLProcessor.__normal_tokens(doc, stem=False), cluster_key)
+            if current_similarity < best_similarity:
+                best_key = cluster_key
```

Comment:
use builtin [`max`](https://docs.python.org/3/library/functions.html#max) function

Reply:
I think a list comprehension utilizing max would make this code less understandable and would be way too long.